File: Java/day13.java
import java.util.Scanner;

public class SumNaturals {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a value: ");
        int n = sc.nextInt();

        if (n <= 0) {
            System.out.println("Only natural numbers are allowed.");
        } else {
            int sum = n * (n + 1) / 2;
            System.out.println("The sum of the first " + n + " natural numbers is: " + sum);
        }
        sc.close();
    }
}

File: Python/day13.py
n = int(input("Enter a value: "))
if n <= 0:
    print("Only natural numbers are allowed.")
else:
    sum = 0
    for i in range(1, n + 1):
        sum += i
    print(f"The sum of the first {n} natural numbers is: {sum}")
